### ISSUE TITLE
A little question

### ISSUE THREAD

**Issue by zqdeepbluesky** [cguindel/eval_kitti#5]
hi, thanks for your code ,I have a little question about lists on your readme,just like you say:
"lists, containing the .txt files with the train/validation splits. These files are expected to contain a list of the used image indices, one per row."
"evaluate_object should be called with the name of the results folder and the validation split; e.g.: ./evaluate_object leaderboard valsplit"
I wonder whether valsplit.txt is refer rotest.txt,cause we split the txt to train.txt, test.txt, trainval.txt, val.txt,so I don't know valsplit.txt Refer to which txt，can you please help me ,thanks so much,have a good day.

**Comment by cguindel** [comment]
Hi @zqdeepbluesky. The name of the file with the image indexes is just an example since the command line arguments should be certainly modified according to your setup. As you can check [here](https://github.com/cguindel/lsi-faster-rcnn/tree/master/data/kitti/lists), I do indeed have a valsplit.txt file, and that is why I use that name in the usage sample. You only have to make sure that you have ground-truth labels for the images that you are using for evaluation.

**Comment by zqdeepbluesky** [comment]
thanks so much for your answer,I will try it on my evaluation labels.

**Comment by DonghoonPark12** [comment]
@cguindel  Hi cguindel. in your example [e.g.: ./evaluate_object leaderboard valsplit] what file is included in 'leaderboard'? 
and you said # of argc is three('data/object/label_2' 'lists' 'results')  but why your # of argc is two?

I think more detail explainnation is needed.

**Comment by cguindel** [comment]
Assuming that you are using the master branch, evaluate_object only needs two arguments: the first one is the name of the experiment (leaderboard in the example) and the second one is the list of images used for validation (e.g., valsplit). The set of .txt files containing the detections for each frame is supposed to be at `build/results/<experiment name>/data`. So, if you want to evaluate results for, let's say, "experiment1" (you can choose the name that you want), make sure that `build/results/experiment1/data` is populated with the .txt files obtained with the detector, one per image in the validation set.

**Comment by DonghoonPark12** [comment]
@cguindel It works really thanks.
